docs/user-guide/upgrade-zos-system.md

```diff
@@ -0,0 +1,90 @@
+ - The keystore directory that contains details about certificates used by Zowe, see [KEYSTORE_DIRECTORY](./installandconfig.md#instance-dir).
+
+There are also two data sets that are created by the Zowe install.
+
+- SZWESAMP containing 10 members.  Five are JCL to configure (and deconfigure Zowe), three are PROCLIB members, one contains PPT entries, one is a PARMLIB member, and one contains console commands.  See [SZWESAMP data set](./install-zowe-zos-convenience-build.md#step-3-choose-a-dataset-hlq-for-the-samplib-and-loadlib)
+- SZWEAUTH is an APF Authorized PDSE load library containing the load modules `ZWESIS01` for the cross memory server and `ZWESAUX` for the auxiliary address space, see [SZWEAUTH data set](./install-zowe-zos-convenience-build.md#step-3-choose-a-dataset-hlq-for-the-samplib-and-loadlib).
+
+
+## Upgrading the instance directory
+
+A Zowe instance directory is created using the script `zowe-configure-instance.sh` contained in the `<RUNTIME_DIR>/bin` directory. 
+
+After installing a new version of Zowe the new runtime will either be in the same directory as the previous version, e.g. `/usr/lpp/zowe` if SMP/E has been used, or else may be in a different directory, e, `~/zowe/zowe-r.M.m` if using a convenience build installation.
+
+In both situations the instance directory used for the previous version of Zowe can be kept and re-used to launch the new version of Zowe.  Run the script `zowe-configure-instance.sh` from the new `<RUNTIME_DIR>/bin` directory with the `-c` argument pointing to the location of the existing instance directory. This is the same method used to create an instance directory with default values in an empty target directory, however if `-c` argument is a pre-existing instance directory rather than wiping and creating fresh contents, the contents are updated.  This includes updating the `manifest.json` file in the instance directory allowing you to see the version that an instance was last configured from.  In addition new values with their defaults may be added to the existing `instance.env` file as new releases of Zowe offer new configuration options.  Options previously changed from their default, such as port values or locations of dependent runtimes are not modified.
+
+The `zowe-configure-instance.sh` script will echo any values that are added to the `instance.env` file.
+
+```
+Missing properties that will be appended to /u/winchj/zowe-instance/instance.env:
+```
+
+A Zowe instance directory `instance.env` file points to the `RUNTIME_DIR` directory used to launch a Zowe instance with the value `ROOT_DIR`.  This directory is **NOT** updated by `zowe-configure-instance.sh` and needs to be updated manually, see [Reviewing the instance.env file](./configure-instance-directory.md#comopnent-prerequisites).
```

Comment:
This isn't correct. Since 1.13 the ROOT_DIR is updated when zowe-configure-instance is run.

Reply:
Excellent !  I'll update the docs to reflect this.  I think at one point it used not to do the update but it now does and I didn't pick up on that.  Sweet !